### ISSUE TITLE
doesn't take in all data for llama 2 7b

### ISSUE THREAD

**Issue by harshpp707** [PromtEngineer/localGPT#659]
I am using llama-2 7b model. I am providing 1 MB file with around 97000 words in it. Is it natural to have this with this much amount of issue?

**Comment by sanket038** [comment]
same with me, I am using CSV and XLSX with same tabular data, it only ingests around 15-20 rows.

**Comment by harshpp707** [comment]
Did you find any solution? One thing I understood from researching is that context length limit is 6k max which is about 3000 words only. Here is the link to it
https://agi-sphere.com/context-length/

**Comment by sanket038** [comment]
Not really, I just found out today morning and still investigating it. Initially I thought it must be someting to do with ingest.py. But it seems altogether a limitation. As an alternate, I am checking
 (https://github.com/imartinez/privateGPT) to see if this limitation is solved there. But installing dependencies is nightmare, stuck since last 4-5 hours. :(
